<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-88BCUOE" failures="2" tests="2" name="Test" time="38.794" errors="0" timestamp="2022-06-08T16:27:07 IST">
  <testcase classname="avactis.testproject.SignInTest" name="logintest" time="6.456">
    <failure type="java.lang.AssertionError" message="Test case failed expected [My Account] but found [Avactis Demo Store]">
      <![CDATA[java.lang.AssertionError: Test case failed expected [My Account] but found [Avactis Demo Store]
at avactis.testproject.SignInTest.logintest(SignInTest.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
  <testcase classname="avactis.testproject.SignInTest" name="logintest" time="4.674">
    <failure type="java.lang.AssertionError" message="Test case failed expected [My Account] but found [Avactis Demo Store]">
      <![CDATA[java.lang.AssertionError: Test case failed expected [My Account] but found [Avactis Demo Store]
at avactis.testproject.SignInTest.logintest(SignInTest.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
</testsuite> <!-- Test -->
